lib/webpack-cli.js

```diff
@@ -162,8 +162,14 @@ class WebpackCLI extends GroupHelper {
         return await require('./commands/external').run(command.name, ...args);
     }
 
-    runHelp() {
+    runHelp(args) {
         const HelpGroup = require('./groups/help');
+        const Commands = ['create', 'loader', 'plugin', 'migrate', 'info', 'serve'];
```

Comment:
We need a better way of extracting these, maybe we can do the same for flags?

Reply:
Would extracting from `cli-flag.js` be fine?

Reply:
Did that in latest commits.